packages/core-extensions/src/moonbase/ui/index.tsx

```diff
@@ -125,9 +128,17 @@ export default (require: typeof WebpackRequire) => {
                   />
                 )}
 
+                {restartNeeded && (
+                  <PanelButton
+                    icon={() => <DangerIcon color={CommonComponents.tokens.colors.STATUS_DANGER} />}
+                    tooltipText="You will need to reload/restart your client for this extension to work properly."
+                  />
+                )}
+
                 <Switch
                   checked={enabled}
                   onChange={() => {
+                    setRestartNeeded(true)
```

Comment:
Instead of this, you could check if `moonlight.enabledExtensions` and `enabled` differ.